tests/ert_tests/services/test_base_service.py

```diff
@@ -301,3 +303,26 @@ def test_local_exec_args_multi():
     assert exec_args[0] == sys.executable
     assert exec_args[2] == "foo"
     assert exec_args[3] == "-bar"
+
+
+def test_cleanup_service_files(tmpdir):
+    global SERVICE_NAMES
+
```

Comment:
Perhaps this global should be deleted or cleaned up so it does not mess up other tests?